packages/flakybot/flakybot.go

```diff
@@ -70,7 +70,24 @@ func main() {
 	log.Println("Sending logs to Flaky Bot...")
 	log.Println("See https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot.")
 
-	if ok := publish(cfg); !ok {
+	logs, err := findLogs(cfg.logsDir)
+	if err != nil {
+		log.Printf("Error searching for logs: %v", err)
+		os.Exit(1)
```

Comment:
non-blocking: consider using different non-zero exit codes for different error scenarios